box-browse-sdk/src/main/java/com/box/androidsdk/browse/service/BoxBrowseController.java

```diff
@@ -115,7 +115,7 @@ private ThumbnailManager createThumbnailManager(BoxSession session) {
             thumbSize = BoxRequestsFile.DownloadThumbnail.SIZE_64;
         }
         try {
-            return mFileApi.getDownloadThumbnailRequest(downloadFile, fileId)
+            return mFileApi.getDownloadThumbnailRequest(downloadFile, fileId, false)
```

Comment:
I think the change to add a new param to getDownloadThumbnailRequest() is not pushed to the content SDK yet correct?